ol-pmtiles v1 import breakage

Hi,

We have an [open PR](https://github.com/bcgov/wps/actions/runs/10894194014/job/30230515616?pr=3926#step:10:241) to bump `ol-pmtiles` from `0.5.0 -> 1.0.0` but our tests break with this error:

```
Error: Cannot find module 'wps/web/node_modules/ol/source/DataTile' imported from wps/web/node_modules/ol-pmtiles/dist/esm/index.js
Did you mean to import "ol/source/DataTile.js"?
```

We run node 20 but I also tried with 18.

`ol`/`ol-pmtiles` combination is:
```
    "ol": "10.0.0",
    "ol-pmtiles": "^1.0.0",
 ```
 
 Also tried bumping `ol`.

--- a/openlayers/examples/raster.html
+++ b/openlayers/examples/raster.html
@@ -4,7 +4,7 @@
         <meta charset="utf-8"/>
         <script src="https://cdn.jsdelivr.net/npm/ol@v9.0.0/dist/ol.js"></script>
         <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/ol@v9.0.0/ol.css">
-        <script src="https://unpkg.com/ol-pmtiles@1.0.0/dist/olpmtiles.js"></script>
+        <script src="https://unpkg.com/ol-pmtiles@1.0.1/dist/olpmtiles.js"></script>
         <style>
             body, #map {
                 height:100vh;

--- a/openlayers/examples/vector.html
+++ b/openlayers/examples/vector.html
@@ -4,7 +4,7 @@
         <meta charset="utf-8"/>
         <script src="https://cdn.jsdelivr.net/npm/ol@v9.0.0/dist/ol.js"></script>
         <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/ol@v9.0.0/ol.css">
-        <script src="https://unpkg.com/ol-pmtiles@1.0.0/dist/olpmtiles.js"></script>
+        <script src="https://unpkg.com/ol-pmtiles@1.0.1/dist/olpmtiles.js"></script>
         <style>
             body, #map {
                 height:100vh;

--- a/openlayers/package.json
+++ b/openlayers/package.json
@@ -1,6 +1,6 @@
 {
   "name": "ol-pmtiles",
-  "version": "1.0.0",
+  "version": "1.0.1",
   "description": "PMTiles sources for OpenLayers",
   "type": "module",
   "main": "dist/cjs/index.cjs",

--- a/openlayers/src/index.ts
+++ b/openlayers/src/index.ts
@@ -1,21 +1,22 @@
-import { type Data } from "ol/DataTile";
+// see import style (.js) at https://openlayers.org/en/latest/examples/data-tiles.html
+import { type Data } from "ol/DataTile.js";
 import {
   type Options as DataTileSourceOptions,
   default as DataTileSource,
-} from "ol/source/DataTile";
-import TileState from "ol/TileState";
-import { MVT } from "ol/format";
-import type TileSource from "ol/source/Tile";
-import { type Extent } from "ol/extent";
-import type Projection from "ol/proj/Projection";
-import type Tile from "ol/Tile";
-import type VectorTile from "ol/VectorTile";
+} from "ol/source/DataTile.js";
+import TileState from "ol/TileState.js";
+import { MVT } from "ol/format.js";
+import type TileSource from "ol/source/Tile.js";
+import { type Extent } from "ol/extent.js";
+import type Projection from "ol/proj/Projection.js";
+import type Tile from "ol/Tile.js";
+import type VectorTile from "ol/VectorTile.js";
 import {
   type Options as VectorTileSourceOptions,
   default as VectorTileSource,
-} from "ol/source/VectorTile";
-import type RenderFeature from "ol/render/Feature";
-import { createXYZ, extentFromProjection } from "ol/tilegrid";
+} from "ol/source/VectorTile.js";
+import type RenderFeature from "ol/render/Feature.js";
+import { createXYZ, extentFromProjection } from "ol/tilegrid.js";
 import { PMTiles, Header, Source } from "pmtiles";
 
 export class PMTilesRasterSource extends DataTileSource {

